### DISCUSSION TITLE
How to Fix this Deprecation error ?

### DISCUSSION THREAD

**Discussion by Nulinz** [gdelataillade/alarm#234] [Q&A] [answered]
C:\Users\myusername\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_fgbg-0.3.0\android\src\main\java\com\ajinasokan\flutter_fgbg\FlutterFGBGPlugin.java uses or overrides a deprecated API.         
Note: Recompile with -Xlint:deprecation for details. 
when use flutter build apk --target-platform android-arm64 --analyze-size  above error kindly give me a solution

**Comment by gdelataillade** [comment, marked as answer]
Hi @Nulinz 

I updated `flutter_fgbg` dependency to version `0.6.0` in the latest plugin version. Let me know if you still have the warning.

**Reply by Nulinz** [reply]
Note: C:\Users\myusername\AppData\Local\Pub\Cache\hosted\pub.dev\flutter_fgbg-0.6.0\android\src\main\java\com\ajinasokan\flutter_fgbg\FlutterFGBGPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

i have the same issue after changing the version

**Reply by gdelataillade** [reply]
Well this is just a warning so it shouldn't affect your app build. Furthermore, the issue comes from the dependency `flutter_fgbg` so there's nothing I can do about it. You can open an issue in their repository if you wish: https://github.com/ajinasokan/flutter_fgbg/issues

**Reply by Nulinz** [reply]
Thanks for Upgrading flutter_fgbg version  it helped lot to build the app!!

**Reply by Nulinz** [reply]
I noticed you've changed the version with the flutter_fgbg plugin. it is impacting foreground and background services in my Flutter app.If so, did you encounter any issues with background services foreground and background states? the alarm not ringing on the lock screen and also after killing the app could you give me the solution for this
I'd really appreciate any insights you can share!
